test/cqlpy/test_secondary_index.py

```
@@ -1942,25 +2176,227 @@ def test_paging_and_drop_index_no_allow_filtering(cql, test_keyspace):
+        # ALLOW FILTERING, so a base-table scan is still a legal fallback.
+        stmt = SimpleStatement(f"SELECT p FROM {table} WHERE v=17 ALLOW FILTERING", fetch_size=page_size)
+        with rest_api.scylla_inject_error(cql, "paging_state_without_query_plan"):
+            r = cql.execute(stmt)
+            assert len(r.current_rows) == page_size
+            assert r.has_more_pages
+            cql.execute(f"DROP INDEX {test_keyspace}.{index_name}")
+            # What the fallback returns is #18992 and not asserted, but it must
+            # not be the refusal a recorded plan produces.
+            try:
+                cql.execute(stmt, paging_state=r.paging_state)
+            except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)